### PR TITLE
[MRG] feat(Localisation): Sets up locales scaffolding and first content

### PR DESCRIPTION
**Pull request by joaobarcia**

### CODE REVIEW

File: netlify.toml [hunk 1]
```diff
@@ -28,3 +28,20 @@
...
+# Redirect to appropriate region and language based on ip location and browser language
+# The redirects engine will process the first matching rule it finds, reading from top to bottom.
+# https://docs.netlify.com/routing/redirects/#syntax-for-the-netlify-configuration-file
+
+# If browser is in German, show CH/DE page
+[[redirects]]
+  from = "/"
+  to = "/ch/de"
+  status = 301
+  conditions = {Language = ["de"]}
```
Comment (betatim) [inline netlify.toml:41]: Do we have to `force = true` these redirects?

Recently Netlify stopped redirecting if there is actual content that would be "hidden" by the redirect. From testing the deploy preview the answer seems to be "No we don't need to force this". Wondering if there is a trap we are about to step into :-/

Comment (joaobarcia) [reply netlify.toml:41]: I'm unsure, so I would vote for following what the tests seem to say is correct and leave it as is, what do you think?

File: pages/index.vue [hunk 2]
```diff
@@ -206,11 +176,10 @@
...
             >
-              <strong
-                >Sie profitieren für begrenzte Zeit vom Vorzugspreis von CHF
-                19.-</strong
-              ><br />
-              (regulärer Preis: 25.-)
+              <p>
+                <strong>{{ $t('payment.price_info1') }}</strong>
+              </p>
+              <p>{{ $t('payment.price_info2') }}</p>
```
Comment (fursinger) [inline pages/index.vue:182]: This should be in the same `<p>` as `_info1` with a `<br>` in between

File: netlify.toml (hunk 1 above)
Comment (betatim) [reply netlify.toml:41]: Nods. experimenting at the moment

**Comment by dkumpf1** [comment]
[DEPLOY-PREVIEW]

Tested: NOT OK

The automatic language re-direction doesn't work for me. Also after "CHF 19" there is a missing " . " for that see:
five_error_message_2

File: pages/index.vue (hunk 2 above)
Comment (fursinger) [reply pages/index.vue:182]: I'll do this later after this PR has been merged

File: netlify.toml (hunk 1 above)
Comment (betatim) [inline netlify.toml:41]:
```suggestion
# If browser is in English, show CH/EN page
[[redirects]]
  from = "/"
  to = "/ch/en"
  status = 301
  conditions = {Language = ["en"]}
```

File: netlify.toml [hunk 3]
```diff
@@ -28,3 +28,20 @@
...
+
+# Otherwise, display CH/EN page
+[[redirects]]
+  from = "/"
+  to = "/ch/en"
```
Comment (betatim) [inline netlify.toml:46]:
```suggestion
# Otherwise, display CH/DE page
[[redirects]]
  from = "/"
  to = "/ch/de"
```

File: locales/en_CH.js [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+      first_name: 'First name',
+      last_name: 'Name',
+      name_is_required: 'Name is required',
+      pay_now: 'Pay now',
+      terms: "Skribble's {0} applies.",
+      terms_linkText: 'privacy policy',
+      terms_linkURL: 'https://www.skribble.com/privacy/',
+      you_must_agree: 'You must agree to start the process.',
+    },
+    price_info1: 'Benefit from the special price of CHF 19– for a limited time',
```
Comment (fursinger) [inline locales/en_CH.js:111]:
```suggestion
    price_info1: 'Benefit from the special price of CHF 19.– for a limited time',
```

**Pull request merged**